velodrome/sql/model.go

```diff
@@ -23,8 +23,8 @@ import (
 
 // Issue is a pull-request or issue. Its format fits into the ORM
 type Issue struct {
-	Repository     string
-	ID             int
+	Repository     string `gorm:"primary_key"`
+	ID             string `gorm:"primary_key"`
```

Comment:
Yeah, I'll open a bug to Gorm. It's dumb, if ID is an int, it will use this as a single field primary key.